app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Admin
+  class CategoriesController < BaseCategoriesController
+    before_action :resource, :model
+
+    private
+    def permitted_params
+      params.require(:category).permit(:title,
+                                       :position,
+                                       :read_only,
+                                       :category_type
+      )
+    end
+
+    def resource
+      :categories
+    end
+
+    def model
+      Category
```

Comment:
I think that `Category` could be used instead of `:categories` as well

Reply:
I tried it but Pundit expects this specific symbol. I can extract it from the @record maybe as ```@record.model_name``` but I have to try first, not sure...